<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>379</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>League Latency 2  [dev0.1a]</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rsrc.qrc">
    <normaloff>:/windIcon/images/icon.ico</normaloff>:/windIcon/images/icon.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="4">
     <widget class="QToolButton" name="toolButton_refresh">
      <property name="toolTip">
       <string>Refresh</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="rsrc.qrc">
        <normaloff>:/refrshIcon/images/refsh.png</normaloff>:/refrshIcon/images/refsh.png</iconset>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEdit_ping_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string extracomment="Shows ping.">Shows your current Ping to the selected server</string>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="maxLength">
       <number>7</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_7">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEdit_ping_6">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string extracomment="Shows ping.">Shows your current Ping to the selected server</string>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="maxLength">
       <number>7</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="9">
     <widget class="PlotWidget" name="plotArea">
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>320</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEdit_ping_5">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string extracomment="Shows ping.">Shows your current Ping to the selected server</string>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="maxLength">
       <number>7</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Total Time [s]</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Avg. Ping [ms]</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Max. Ping [ms]</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="pushButton_pi">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="rsrc.qrc">
        <normaloff>:/currPingIcons/images/stgd.png</normaloff>:/currPingIcons/images/stgd.png</iconset>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEdit_time">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string extracomment="Shows ping.">Shows the total time elapsed</string>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="maxLength">
       <number>6</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Curr. Ping [ms]</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Min. Ping [ms]</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEdit_ping_4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string extracomment="Shows ping.">Shows your current Ping to the selected server</string>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="maxLength">
       <number>7</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEdit_ping">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string extracomment="Shows ping.">Shows your current Ping to the selected server</string>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="maxLength">
       <number>7</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEdit_ping_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string extracomment="Shows ping.">Shows your current Ping to the selected server</string>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="maxLength">
       <number>7</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QToolButton" name="toolButton">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Add a server</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="rsrc.qrc">
        <normaloff>:/addIcon/images/add.png</normaloff>:/addIcon/images/add.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="pushButton_pi2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="rsrc.qrc">
        <normaloff>:/avgPingIcons/images/null.png</normaloff>:/avgPingIcons/images/null.png</iconset>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="3">
     <widget class="QComboBox" name="comboBox">
      <property name="editable">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>North America</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Latin America North</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QPushButton" name="pushButton_quit">
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="pushButton_pause">
      <property name="toolTip">
       <string>Pause the graph</string>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>775</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionCheck_for_Update"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <widget class="QMenu" name="menuLL_Help">
     <property name="title">
      <string>LL Help...</string>
     </property>
     <property name="icon">
      <iconset resource="rsrc.qrc">
       <normaloff>:/menuBarIcons/images/help.png</normaloff>:/menuBarIcons/images/help.png</iconset>
     </property>
     <addaction name="actionShortcuts"/>
     <addaction name="actionDoumentation"/>
    </widget>
    <addaction name="menuLL_Help"/>
    <addaction name="actionExtra"/>
    <addaction name="separator"/>
    <addaction name="actionVersion"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_LL"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/menuBarIcons/images/quit.png</normaloff>:/menuBarIcons/images/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/menuBarIcons/images/settings.png</normaloff>:/menuBarIcons/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionCheck_for_Update">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/menuBarIcons/images/update.png</normaloff>:/menuBarIcons/images/update.png</iconset>
   </property>
   <property name="text">
    <string>Check for Update</string>
   </property>
  </action>
  <action name="actionExtra">
   <property name="text">
    <string>Extra</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/menuBarIcons/images/version.png</normaloff>:/menuBarIcons/images/version.png</iconset>
   </property>
   <property name="text">
    <string>Version</string>
   </property>
  </action>
  <action name="actionAbout_LL">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/menuBarIcons/images/about.png</normaloff>:/menuBarIcons/images/about.png</iconset>
   </property>
   <property name="text">
    <string>About LL</string>
   </property>
  </action>
  <action name="actionShortcuts">
   <property name="text">
    <string>Shortcuts</string>
   </property>
  </action>
  <action name="actionDoumentation">
   <property name="text">
    <string>Doumentation</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/menuBarIcons/images/load.png</normaloff>:/menuBarIcons/images/load.png</iconset>
   </property>
   <property name="text">
    <string>Load...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="rsrc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_quit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1075</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
